server/game/cards/02-AoA/SirMarrows.js

```diff
@@ -4,17 +4,21 @@ class SirMarrows extends Card {
     setupCardAbilities(ability) {
         this.reaction({
             when: {
-                onReap: (event) =>
-                    event.card.controller !== this.controller && event.card.type === 'creature'
+                onModifyAmber: (event, context) =>
+                    event.reap && event.player !== context.source.controller
             },
             gameAction: ability.actions.sequential([
-                ability.actions.placeAmber({
-                    amount: 1,
-                    target: this
-                }),
-                ability.actions.loseAmber((context) => ({
-                    target: context.player.opponent,
-                    amount: 1
+                ability.actions.capture((context) => ({
+                    amount: context.event.amount || 0
```

Comment:
Why the || 0? Can this ever be undefined?

Reply:
I don't think it can, just being over careful, I will remove it.